### PR TITLE
release/21.03: make exports synchronous again

### PR DESCRIPTION
**Pull request by ajeetdsouza**
21.03.0 has async backups, but not exports. An earlier cherry-pick moved the async backups onto the async Task framework, but also made exports async, which breaks backwards compatibility.

This PR is a temporary fix to make exports sync just for 21.03.1.

---
This change is [Reviewable](https://reviewable.io/reviews/dgraph-io/dgraph/7877)

### CODE REVIEW

**Comment by ajeetdsouza** [comment]
CI has failed due to a bug on `dgus-hotel`. I've rerun CI here, and it has passed: https://teamcity.dgraph.io/buildConfiguration/Dgraph_Ci/139278

File: dgraph/cmd/alpha/admin.go [hunk 1]
```diff
@@ -158,6 +160,46 @@ func shutDownHandler(w http.ResponseWriter, r *http.Request) {
...
+	gqlReq := &schema.Request{
+		Query: `
+		mutation export($format: String) {
+		  export(input: {format: $format}) {
+			response {
+			  code
+			}
+		  }
+		}`,
+		Variables: map[string]interface{}{},
```
Comment (jarifibrahim) [inline dgraph/cmd/alpha/admin.go:192]: @ajeetdsouza Don't you need to pass the `format` variable here?

File: systest/export/export_test.go [hunk 2]
```diff
@@ -53,17 +53,21 @@ func TestExportSchemaToMinio(t *testing.T) {
...
-	schemaFile := ""
-	doneCh := make(chan struct{})
-	defer close(doneCh)
-	for obj := range mc.ListObjectsV2(bucketName, "dgraph.", true, doneCh) {
-		if strings.Contains(obj.Key, ".schema.gz") {
-			schemaFile = obj.Key
-		}
+	result := requestExport(t, minioDest, "rdf")
+
+	log.Printf("ajeet %+v", result)
```
Comment (ahsanbarkati) [inline systest/export/export_test.go:58]: Do we want to remove this log?

Comment (ajeetdsouza) [reply systest/export/export_test.go:58]: Fixed.

File: dgraph/cmd/alpha/admin_backup.go [hunk 3]
```diff
@@ -0,0 +1,68 @@
+// +build !oss
+
+/*
+ * Copyright 2018 Dgraph Labs, Inc. and Contributors
```
Comment (jarifibrahim) [inline dgraph/cmd/alpha/admin_backup.go:4]: 2021?

**Review by jarifibrahim — APPROVED** [review]
Looks good to me.

**Pull request merged**